include/odp/api/spec/traffic_mngr.h

```diff
@@ -252,6 +276,15 @@ typedef struct {
 	/** tm_node_threshold indicates that the tm_nodes at this
 	 * level support threshold profiles. */
 	odp_bool_t tm_node_threshold;
+
+	/** tm_node_stats indicates the statistics supported by nodes at this level.
+	 * When True, that stat counter is supported, while False indicates it is not
+	 * supported.
+	 *
+	 * @see odp_tm_node_stats_type_t
+	 * @see odp_tm_node_stats()
+	 */
+	odp_bool_t tm_node_stats[ODP_TM_NODE_STATS_MAX];
```

Comment:
I think that in addition to having capabilities, maybe it should be configurable (in the per-level requirements or node parameters) which counters, if any, need to be counted.

Reply:
Ok, In our HW, it will be a no-op as the stats are counted in HW and no option to enable/disable it. We are fine to have such capability if needed.